backend/data_import/pipeline/catalog.py

```diff
@@ -453,3 +459,25 @@ def register(cls, option: Option):
         file=SPEECH_TO_TEXT_DIR / "audio_files.txt",
     )
 )
+
+# Article Annotation
+Options.register(
+    Option(
+        display_name=JSON.name,
+        task_id=ARTICLE_ANNOTATION,
+        file_format=JSON,
+        arg=ArgColumn,
+        file=ARTICLE_ANNOTATION_DIR / "example.json"
+    )
+)
+
+# Custom Relation Extraction
+Options.register(
+    Option(
+        display_name="JSONL(Relation)",
+        task_id=CUSTOM_RELATION_EXTRACTION,
+        file_format=JSONL,
```

Comment:
I see that because we want to make it "compatible" with the previously worked Custom Relation Extraction, we keep this registered option using JSONL.

However, I am wondering if it is possible to register one more option, "JSON(Relation)". So that we will have three options in total: JSON, JSON(Relation), and JSONL(Relation). I remember vaguely that there was a problem using json instead of jsonl for importing dataset with relation, but I don't remember exactly what the problem was. XD

Reply:
Yeah I remember there was related problem also XD but I added the JSON(Relation) and it works, the spans and relations are imported, and categories also (only categories are not showing up on UI since we havent adjusted the UI :smile: )